src/signature.ts

```diff
@@ -96,27 +96,61 @@ export const toRpcSig = function(v: BNLike, r: Buffer, s: Buffer, chainId?: BNLi
   return bufferToHex(Buffer.concat([setLengthLeft(r, 32), setLengthLeft(s, 32), toBuffer(v)]))
 }
 
+/**
+ * Convert signature parameters into the format of Compact Signature Representation (EIP-2098).
+ * @returns Signature
+ */
+ export const toRpcSigCompact = function(v: BNLike, r: Buffer, s: Buffer, chainId?: BNLike): string {
```

Comment:
I do not think this is related to the RPC, or does any `eth_` RPC method support this now?

If not, I'd suggest something like `toEIP2098CompactSig`, `toEIP2098Sig`, or `toCompactSig` (least preferred is the last as the proposal may not be widespread enough to have "compact sig" necessarily connect people with the EIP)